doc/source/whatsnew/v1.5.0.rst

```diff
@@ -150,6 +150,7 @@ Other enhancements
 - Added ``validate`` argument to :meth:`DataFrame.join` (:issue:`46622`)
 - A :class:`errors.PerformanceWarning` is now thrown when using ``string[pyarrow]`` dtype with methods that don't dispatch to ``pyarrow.compute`` methods (:issue:`42613`)
 - Added ``numeric_only`` argument to :meth:`Resampler.sum`, :meth:`Resampler.prod`, :meth:`Resampler.min`, :meth:`Resampler.max`, :meth:`Resampler.first`, and :meth:`Resampler.last` (:issue:`46442`)
+- Moved :class:`DataError` from ``pandas.core.base`` to ``pandas.errors`` (:issue:`27656`)
```

Comment:
this is not necessary as there isn't a visible user change (pandas.core is considered private). so you can re-word to say its now exposed in `pandas.errors` or just omit the note is ok too.

Reply:
Thank you for the feedback, I've reworded the note